examples/openbankapi.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Open Bank Project
+ * 
+ * Example of retrieving an Oauth v1.0 token from Open Bank Project Oauth server
+ *
+ *
+ * @author      Amir Duran <amir.duran@gmail.com>
+ * @license     http://www.opensource.org/licenses/mit-license.html  MIT License
+ */
+
+use OAuth\OAuth1\Service\OpenBankProject;
+use OAuth\Common\Storage\Session;
+use OAuth\Common\Consumer\Credentials;
+use OAuth\OAuth1\Signature\Signature;
+/**
+ * Bootstrap the example
+ */
+require_once __DIR__ . '/bootstrap.php';
+
+// Session storage, for testing purposes I choose Session. You can extend TokenStorageInterface and make connection with DB
+$storage = new Session();
+
+// Setup the credentials for the requests
+$credentials = new Credentials(
+    $servicesCredentials['openBankProject']['key'],
+    $servicesCredentials['openBankProject']['secret'],
+    'http://localhost/oauth/oAuthPHPLibrary/examples/openbankapi.php'

```

Comment:
I will change this to PUT YOUR LOCAL PATH TO THE PROJECT, so someone understand what to do.